app/views/core_induction_programme/lesson_parts/show_delete.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= govuk_breadcrumbs breadcrumbs: course_lesson_breadcrumbs(@current_user, @course_lesson_part.course_lesson) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h2>Confirm you would like to remove <i><%= @course_lesson_part.title %></i> from <%= @course_lesson_part.course_lesson.title %>.</h2>
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <div class="govuk-warning-text__text">This lesson part can not be retrieved once it has been deleted.</div>
+      </div>
+    <p>You can return to <b><%= @course_lesson_part.course_lesson.title %></b> without making any changes by clicking cancel</p>
```

Comment:
Something something `div` has wrong indentation :)

Reply:
Any particular reason you're using `<b>` over `<strong>`?

Reply:
I forgot about strong to be honest but reading about differences <b> is used for drawing attention and with <strong> its to indicate content is of greater importance. I think <b> would be more  as it's just trying to clear what lesson it is the user is working on. 

Happy to go with either though, i didn't consider between the two initially.